### ISSUE TITLE
Battery widget

### ISSUE THREAD

**Issue by WizenedPottle** [end-4/dots-hyprland#18]
I'm a newbie with eww, but I'm trying to figure it out. I just want to make it clear from the start.

The battery widget isn't working for me. After some investigation, I discovered that my battery information is stored in `/sys/class/power_supply/BAT0` compared to `BATT` in your configuration. I :s///g'ed that, and now the script does what it's supposed to do. However, the widget still doesn't work. Can you recommend my next steps in figuring this out?

Great repo btw!

**Comment by end-4** [comment]
hi there, 
can you try manually running `cd .config/eww && scripts/battery` and provide the output? does it output a proper json?

**Comment by end-4** [comment]
- the json looks alright
- now, how does the battery indicator show up for you? 
  - does the circular progress show up? 
  - does the percentage next to it display properly? 
  
would be nice if you could send a screenshot

**Comment by WizenedPottle** [comment]
Apologies for the delay in getting back to you. I had an exam in Thermodynamics, which required my full attention.

Regarding the issue, I am using your main repository but have customized the color scheme using Summer Gruve. Here is a screenshot: 
![](https://github.com/end-4/dots-hyprland/assets/94008053/ce53db73-b51b-4360-a778-23aad894efc8)

Please let me know if you have any insights or suggestions for resolving this issue. Thank you for your understanding.

**Comment by end-4** [comment]
Oops i think my commit saying `hopefully fixes #18` auto-closed it lol
Anyway you should try again with that fix in https://github.com/end-4/dots-hyprland/commit/b05be1394b2230ce98c278d2c773aeac8183da47
- If it still doesnt work, run `eww get EWW_BATTERY` and see if it outputs proper values. Should be something like this 
```json
{"BATT": { "status": "Discharging", "capacity": 25 }, "total_avg": 26.0}
```

**Comment by WizenedPottle** [comment]
Thank you man!
